### PR TITLE
add a new benchmark "BHE Array" to the web content.

### PR DESCRIPTION
**Pull request by HBShaoUFZ**
New benchmark description. Try Jenkins to see the effect. Data upload not fully completed yet. Please do not merge.

### CODE REVIEW

**Comment by endJunction** [comment]
@bilke As touched before: In the [web](https://jenkins.opengeosys.org/job/ufz/job/ogs/job/PR-2084/4/Web/) the images are not shown...

**Comment by bilke** [comment]
@HBShaoUFZ Now the images are fine! For the formulas I am working on a solution..

**Comment by HBShaoUFZ** [comment]
@endJunction From my point of view, the PR is ready for review.

File: ProcessLib/HeatConduction/Tests.cmake [hunk 1]
```diff
@@ -42,3 +42,15 @@ AddTest(
...
+# The 25 BHE array benchmark
+# test results are compared to 2D simulation result
+AddTest(
+        NAME BHE_Array_2D
+        PATH Parabolic/T/2D_BHE_array
+        EXECUTABLE ogs
+        EXECUTABLE_ARGS bhe2d.prj
+        TESTER vtkdiff
+        DIFF_DATA
+        standard_solution_bhe2d_pcs_0_ts_840_t_72576000.000000.vtu bhe2d_pcs_0_ts_840_t_72576000.000000.vtu temperature temperature 1e-4 1e-4
```
Comment (endJunction) [inline ProcessLib/HeatConduction/Tests.cmake]: Not sure if the absolute and relative errors of 1e-4 aren't too sloppy. I'd expect something in range 1e-10 to 1e-15 for a benchmark. Of course, if temperature is crossing 0, then the relative error does not make sense and should be set to 0.

**Comment by HBShaoUFZ** [comment]
@endJunction As we have discussed, the spatial graph and the absolute error plot have been added. From my opinion, this PR is ready for merge.

**Pull request merged**